securedrop/sass/source.sass

```diff
@@ -255,6 +255,11 @@ p#max-file-size
   font-style: italic
   margin-bottom: 0
 
+p#min-msg-length
+  font-size: 10px
+  font-style: italic
+  margin-bottom: 0
```

Comment:
I think this margin could be set to 12px - `#max-file-size` after which this is modeled does not have a bottom margin, but it's wrapped in an element with a 12px padding all around, which isn't really suitable for the `textarea` side of things

Reply:
as you have spoken, so has it been actioned.